static/app/components/compactSelect/composite.tsx

```diff
@@ -17,27 +17,33 @@ interface BaseCompositeSelectRegion<Value extends React.Key> {
 
 /**
  * A single-selection (only one option can be selected at a time) "region" inside a
- * composite select. Each "region" is a separated, self-contained select box (each
- * renders as a list box with its own list state) whose selection values don't interfere
+ * composite select. Each "region" is a separated, self-contained selectable list (each
+ * renders as a `ul` with its own list state) whose selection values don't interfere
```

Comment:
Just a ton of renaming/rephrasing since "list" can now be either a grid list or a list box.